examples/basic/tidb-monitor.yaml

```diff
@@ -17,4 +17,9 @@ spec:
   reloader:
     baseImage: pingcap/tidb-monitor-reloader
     version: v1.0.1
+  dmClusters:
+  - name: basic
+  dmInitializer:
+    baseImage: hub.pingcap.net/chenbin_test/dm-monitor-initializer
+    version: v2.0.0
```

Comment:
I guess If the `dmClusters` name is different from the `cluster` name, the tidbmonitor will failed to scrape metrics from it, will it?

Reply:
yes, i will fix it soon

Reply:
> I have a question, is it necessary to set different names for tidbmonitor and dmmonitor?

I'm not familiar with `DM` so I can't answer this question. But I think the `TidbMonitor` will try to scrape multiple DMClusters according to the `Cluster` while the only single `DMCluster`  is defined in spec.

Moreover, I think the current implementation couldn't handle the situation while the users want to use `DM` independetly which is serving an outside Kubernetes TiDB

Reply:
> > I have a question, is it necessary to set different names for tidbmonitor and dmmonitor?
> 
> I'm not familiar with `DM` so I can't answer this question. But I think the `TidbMonitor` will try to scrape multiple DMClusters according to the `Cluster` while the only single `DMCluster` is defined in spec.
> 
> Moreover, I think the current implementation couldn't handle the situation while the users want to use `DM` independetly which is serving an outside Kubernetes TiDB

Currently, `DM` won't start an independent discovery pod and rely on an existing tidbcluster's discovery service. In the future, if we find it's necessary to support deploying DM independently we can file another PR to do this.

Reply:
@Yisaer We have fix the problem that `dmClusters` name should be same as `cluster` name. Now `dmClusters` name can be different with `cluster` name.